openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -13,40 +13,40 @@ <h2 class="title">
     </h2>
 
     <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__response">
-        <div class="ui-staff ui-toggle-visibility is--collapsed">
-            <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+        <button class="ui-staff ui-slidable" aria-expanded="false" id="learner_response_{{ submission.uuid }}">
```

Comment:
:+1: Thanks for the tip about `?w=1`, that made the diff way easier to see at a glance.

I also found this chrome plugin useful for wide files like the HTML in this PR, in case anyone else gets annoyed at left-right scrolling repeatedly.
https://chrome.google.com/webstore/detail/wide-github/kaalofacklcidaampbokdplbklpeldpj